### ISSUE TITLE
Allow users to set default heartbeat & retry times

### ISSUE THREAD

**Issue by tomlawesome** [louislam/uptime-kuma#1004]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Everytime you add a new monitor, you must change the default tie to something else unless you want it at 60s with no retries. For example, I like to use 180s heartbeat, 3 retries, 30s apart 

### ✔️ Solution

In settings, allow a field to change the default value for heartbeart, retries and retry time. 

### ❓ Alternatives

There's no alternative.

**Comment by crbon** [comment]
This feature would be really handy for those with custom preferences

**Comment by CommanderStorm** [comment]
What is the difference between cloning+grouping monitors and what you requested.
I think this functionality is already here, but in an slightly different formfactor

**Comment by liuxy0551** [comment]
### 1. Backup
``` bash
docker exec uptime-kuma \
  cp /app/data/kuma.db /app/data/kuma.db.bak
```

### 2. Modify all monitors to 300 seconds
``` bash
docker exec uptime-kuma \
  sqlite3 /app/data/kuma.db \
  'UPDATE monitor SET interval = 300;'
```

### 3. Restart to reload the timers for all monitors
``` bash
docker restart uptime-kuma
```

**Comment by amityweb** [comment]
And if you don’t use docker? I guess just remove the docker exec part?

Anyway this request would be t9 have it in a setting easily changed not cli. 

My suggestion putting the settings into groups would be best I think.

**Comment by CommanderStorm** [comment]
No, above suggestion was how to CHANGE all monitors, not change the default.
